optimum/exporters/openvino/__main__.py

```diff
@@ -410,10 +410,10 @@ class StoreAttr(object):
                     )
                 model.config.pad_token_id = pad_token_id
 
-        if hasattr(model.config, "export_model_type"):
+        if hasattr(model.config, "export_model_type") and model.config.export_model_type is not None:
```

Comment:
why would it be there and None ?

Reply:
FrozenDict behavior returns None if field does not exists in config and does not allow to set any additional fields

Reply:
removed